acasclient/acasclient.py

```diff
@@ -1702,4 +1723,21 @@ def update_cmpdreg_vendor(self, vendor: dict):
     def delete_cmpdreg_vendor(self, id: int) -> bool:
         resp = self.session.delete("{}/api/cmpdRegAdmin/vendors/{}".format(self.url, id))
         resp.raise_for_status()
-        return True
\ No newline at end of file
+        return True
+
+    def setup_items(self, item_type, items):
+        """Create or update items of a given typeKind 
+           ACAS Admin role for this operation
+
+        Args:
+            item_type (str): Type of item to create or update
+            items (list): List of items to create or update
+        """
+        allowed_types = ['experimenttypes', 'experimentkinds', 'statetypes', 'statekinds', 'valuetypes', 'valuekinds',
+                         'labeltypes', 'labelkinds', 'ddicttypes', 'ddictkinds', 'codetables','labelsequences', 'roletypes',
+                         'rolekinds', 'lsroles']
+        if item_type not in allowed_types:
+            raise ValueError("item_type must be one of {}".format(allowed_types))
+        resp = self.session.post("{}/api/setup/{}".format(self.url, item_type), json=items)
+        resp.raise_for_status()
+        return resp.json()
```

Comment:
Added this function because in the tests we want to be able to add a project and then add project roles to that project.  We use this route when syncing roles when saml auth and sync roles is active.